### ISSUE TITLE
feat: New feature for YouTube

### ISSUE THREAD

**Issue by Thanasis-Koligliatis** [revanced/revanced-patches#2486]
### Application

YouTube

### Issue

Watch a video from a playlist.
At the end of the video, stop.
Don't play next video automatically.

### Patch

Watch a video from a playlist.
At the end of the video, stop.
Don't play next video automatically.

### Motivation

As it is now, 
at the end of the video,
you may want to see the comments, or like the video,
but you can't, 
because the next video is playing.

Thank you

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by EhabxAlaa** [comment]
Yes please, i always have to  not forget to  pause the video if it was in a playlist at the end of it so it doesn't automatically go to the next video when i want to read the comments or check discribtion , so this patch would be really useful
